README.md

```
@@ -1,3 +1,22 @@
+### Using a Custom Font (via URL)
+
+You can render quotes with your own font by passing a *publicly accessible* font file URL.
+
+*Parameter:* fontUrl  
+*Supported formats:* .woff2, .woff, .ttf
+
+*Example — Google Fonts (Roboto):*
+![Quote](https://github-readme-quotes-bay.vercel.app/quote?theme=dark&font=Roboto)
+
+### Using a Custom Font (via URL)
```

markdownlint-cli2 (0.17.2)

Multiple headings with the same content

(MD024, no-duplicate-heading)